canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.jsx

```diff
@@ -115,12 +115,14 @@ class RadiosetControl extends React.Component {
 	}
 
 	handleChange(evt) {
-		const oldVal = this.props.controller.getPropertyValue(this.props.propertyId);
-		const newVal = this.convertTargetValue(evt);
-		this.props.controller.updatePropertyValue(this.props.propertyId, newVal);
+		if (!this.props.control.readOnly) {
```

Comment:
Why do we need this if this should already be readOnly?

Reply:
I have included this condition to disable switching between radio button. Please refer below recording in which we can switch between radio buttons even if readOnly is enabled.


https://github.com/user-attachments/assets/e2ba859c-1414-43ee-8af5-48443af3dcf1